CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+
+## Welcome to the MyLibreLab project!
+
+First off, thank you for considering contributing to MyLibreLab. It's people like you that make open source possible! :tada:
+
+MyLibreLab is an open source project and we love to receive contributions from our community — you! There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into MyLibreLab itself.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+# Ground Rules
+
+Responsibilities
+* Ensure that code that goes into core meets all requirements in this checklist: {{CHECKLIST}}
```

Comment:
Also, here we should have some requirements for our code quality/standards

Reply:
something like this maybe:
1. Every PR has to be reviewed before merging.
2. Every addition to the codebase has to be tested.
3. Software design principles like SOLID and patterns

Reply:
If you include checkboxes in the PR template then you can require them to be checked for the PR to be able to be merged. At least forces people to read those rules.

Reply:
Yup! ^^
Feel free to add changes you would like My main focus is on clean commits and PR's, with enforced use of sonarlint and autostyle. So maybe just templates for everything, and we are good for now?